src/wombats/components/pedestal.clj

```diff
@@ -26,9 +26,7 @@
      ::http/file-path "/public"
      ::http/allowed-origins (if (contains? #{:dev :dev-ddb} env)
                               (fn [origin] true)
-                              (fn [origin]
-                                (or (re-matches #".wombats.io" origin)
-                                    (re-matches #"//wombats.io" origin))))
+                              #(re-find #"\.wombats\.io|\/\/wombats.io" %))
```

Comment:
Also we need to make sure both sides of the `|` end with a `$`, otherwise a string like this would pass:

```
wombats.iohackerdomain.com
```